Services/Interfaces/web_interface/templates/onboarding.html

```diff
@@ -0,0 +1,149 @@
+                        data-url="{{url_for('profile', select=profile.profile_id, next=url_for('onboarding'))}}">
+                    <i class="fa fa-check" no-activation-click="true"></i>
+                </button>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+</div>
+{%- endmacro %}
+
+{% macro profile_details_modal(profile) %}
+    <div class="modal fade" id="{{profile.profile_id}}Modal" tabindex="-1" role="dialog" aria-labelledby="#{{profile.profile_id}}ModalLabel" aria-hidden="true">
+      <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+        <div class="modal-content modal-text bg-dark text-light mt-4">
+          <div class="modal-header primary-text">
+              <h5 class="modal-title" id="{{profile.profile_id}}ModalLabel">
+                  {{profile.name}}
+              </h5>
+            <button type="button" class="close text-light" data-dismiss="modal" aria-label="Close">
+              <span aria-hidden="true">&times;</span>
+            </button>
+          </div>
+          <div class="modal-body">
+            {{ profiles_macros.profile_details(profile, profiles_tentacles_details[profile.profile_id], strategy_config,
+              evaluator_config, get_profile_traded_pairs_by_currency, get_profile_exchanges,
+              get_enabled_trader, get_filtered_list, True) }}
+          </div>
+        </div>
+      </div>
+    </div>
+{% endmacro %}
+
+{% block body %}
+<br>
+<div class="card">
+    <div class="card-header">
+        <h2>Welcome to OctoBot</h2>
+    </div>
+    <div class="card-body px-3">
+        {% if current_logged_in_email %}
+        <div class="alert alert-dark row py-0">
+            <div class="col-8 my-auto">
+                Logged in as {{current_logged_in_email}}
+                {% if selected_user_bot["name"] %}
+                    using bot <span class="badge badge-info">{{ selected_user_bot["name"] }}</span>
+                {% else %}
+                    <span class="badge badge-danger">without selected bot</span>
+                {% endif %}
+            </div>
+            {% if can_logout %}
+            <div class="col-4 text-right">
+                <a class="align-right btn btn-sm btn-outline-info waves-effect"
+                   href="{{ url_for('community_logout', next=url_for('onboarding'))}}">
+                    logout
+                </a>
+            </div>
+            {% endif %}
+        </div>
+        {{ m_flash_messages.flash_messages() }}
+        {% else %}
+        <div>
+            Login to you OctoBot Account to access your subscribed profiles as well as the additional profiles.
+
+            <div class="login_box mx-auto mt-1 mt-xl-5">
+                {{ login.login_form(form, is_in_stating_community_env, url_for("onboarding"), "sync_account") }}
+            </div>
+        </div>
+        {% endif %}
+        <h3>
+            Select the profile your OctoBot use
```

Comment:
Should we hide this part while the user is not connected?

Reply:
if we want to force connection on self hosted bots then we should, for now connecting is making subscribed profiles available on self hosted bots. I think it's better to make it possible not to login on self hosted bots